libenkf/include/ert/enkf/ert_run_context.h

```diff
@@ -46,14 +46,14 @@ typedef struct ert_run_context_struct ert_run_context_type;
                                                enkf_fs_type * target_update_fs ,
                                                bool_vector_type * iactive ,
                                                path_fmt_type * runpath_fmt ,
-					       const char * jobname_fmt,
+                 const char * jobname_fmt,
```

Comment:
If I were the maintainer I'd hate you :smile:

Reply:
yes - I can see that; I had some editor f..up with tabs and space, tried to fix it, but some obviously fell through - will fix it.